app/views/layouts/application.html.erb

```diff
@@ -12,10 +12,10 @@
 
   <body>
     <header>
```

Comment:
I would suggest having a class name header for the header, but we will work on style more.